src/libs/DateUtils.ts

```diff
@@ -730,6 +730,26 @@ function formatToSupportedTimezone(timezoneInput: Timezone): Timezone {
     };
 }
 
+/**
+ * Returns the latest business day of input date month
+ *
+ * param {Date} inputDate
+ * returns {number}
+ */
+function getLastBusinessDayOfMonth(inputDate: Date): number {
+    const currentDate = new Date(inputDate);
+
+    // Set the date to the last day of the month
+    currentDate.setMonth(currentDate.getMonth() + 1, 0);
```

Comment:
We can make it easier with date-fns with using just endOfMonth() function.

Reply:
const lastDayOfMonth = endOfMonth(inputDate);

Reply:
Done - 453de73.